### PR TITLE
FEAT: add routing / auth comp

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
### Pages with material ui
- login page
- sigup page  

### Feat using Supabase
- auth Functionality

Signup with valid email  >  confirm email  > login with same id pass
check console for login/signup

### CODE REVIEW

File: src/pages/Auth/login.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import TextField from "@mui/material/TextField";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { loginHandleSubmit } from ".";
```
Comment (Chiragmodi01) [inline src/pages/Auth/login.jsx:14]: Would suggest keeping import in separate files, Like TextField you might use later somewhere else in your code also.

File: src/pages/Auth/signup.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { SignupHandleSubmit } from ".";
+import { Link } from "react-router-dom";
```
Comment (Chiragmodi01) [inline src/pages/Auth/signup.jsx:15]: As mentioned above, use barrel imports

**Review by Chiragmodi01 — COMMENTED** [review]
Nothing much to add.
Pretty concise code. UI looks neat.

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (dusariudaykumar) [inline src/App.jsx:1]: You can remove this because this is not required in the react latest version

File: src/pages/Auth/index.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        password: data.get("password"),
+      });
+      console.log("user", user);
+      console.log("session", session);
+      console.log("error", error);
+    } catch (error) {
+      console.error("something went wrong");
+    } finally {
+      console.log("all went well");
+
```
Comment (dusariudaykumar) [inline src/pages/Auth/index.js:18]: Nice use of finally block

**Review by dusariudaykumar — APPROVED** [review]
The UI and directory structure looks great 👌
Great work!

**Pull request merged**